packages/@react-native-windows/cli/src/index.ts

```diff
@@ -16,6 +17,11 @@ import {runWindowsCommand} from './runWindows/runWindows';
 import {dependencyConfigWindows} from './config/dependencyConfig';
 import {projectConfigWindows} from './config/projectConfig';
 
+import * as appInsights from 'applicationinsights';
+
+appInsights.setup('a7b9ed40-e2a4-4166-bf80-230540f4dcff').start();
```

Comment:
If it's not already there, could we attach package version to the payload we add? We can otherwise have failures from local development pollute telemetry.